### PR TITLE
Honeybee_infORventPerArea Calculator

### PR DESCRIPTION
**Pull request by ayezioro**
The component now allows to calculate infORventPerArea according to ambient conditions or @50Pascal pressure (for PassiveHouse purposes)

### CODE REVIEW

**Comment by mostaphaRoudsari** [comment]
@ayezioro there is an extra underline in ghenv.Component.Name which resulted in not overwriting the old component but creating a new one! Can you get that fix and send a new pull request?

File: src/Honeybee_infORventPerArea Calculator.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    
+    Args:
+        _HBZones: Honeybee zones for which you want to calculate the infiltration or ventilation rates.
+        _airChangeHour: Air Changes per Hour. Given at ambien pressure. Give a value of 0.2 or higher.
+        _at50PA_: Boolean. Set to true if you want to get a value at 50 Pascal. You probably want this for PassiveHouse calculations. Default is False as for ambient calculation.
+    Returns:
+        readMe!: Report of the calculations
+        infORventPerArea: infiltrationRatePerArea or ventilationPerArea in m3/s-m2 (Cubic meters per second per square meter of floor)
+"""
+ghenv.Component.Name = "Honeybee_infORventPerArea Calculator"
```
Comment (mostaphaRoudsari) [inline src/Honeybee_infORventPerArea Calculator.py]: This should change to ```ghenv.Component.Name = "Honeybee infORventPerArea Calculator"``` no underline between Honeybee and infORventPerArea

Comment (ayezioro) [inline src/Honeybee_infORventPerArea Calculator.py]: I know. It was on purpose to be consistent with MOST of the HB names. Most of them are HB_ and just a few are without the underscore.
I think this avoids confusion when looking for it (at least this happened to me).
What do you say?
-A.

Comment (mostaphaRoudsari) [inline src/Honeybee_infORventPerArea Calculator.py]: Make sense. Then we need to remove the older component. I think I can take care of that. Thanks!

**Pull request merged**